### DISCUSSION TITLE
Early access links don't work

### DISCUSSION THREAD

**Discussion by JonathanBout** [community/community#139165] [Copilot Conversations]
Whenever I open an early access link from for example the GitHub blog (like here: https://github.blog/news-insights/product-news/try-out-openai-o1-in-github-copilot-and-models/ the first link)  and click 'join waitlist' I'm redirected to the mobile app and get an error:

![the text 'something went wrong' and a button 'Try Again'](https://github.com/user-attachments/assets/4b9c66ba-41c0-41e5-b108-c38425d09fcc)

Shouldn't the app notice it doesn't have that page and open it in the browser?

**Comment by Akash1134** [comment]
Hi @JonathanBout ,

Thanks for sharing your feedback. The link should direct you to the [sign up page](https://github.com/o1-waitlist-signup?utm_campaign=GitHub_Blog) on a browser. We will share the feedback with the concerned team.

**Reply by JonathanBout** [reply]
Thank you for the quick response!

By manually pasting the link in the browser I indeed ended up on the right page. The link you shared also tried opening in the app and results in the error page
